src/c2goto/library/libm/nextafter.c

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Did we get this implementation from somewhere? Shall we mention it here?

Reply:
From me ;) I'll add a header to the file.